src/test/java/com/trello/trello/stepdefs/LoginSteps.java

```diff
@@ -0,0 +1,87 @@
+package com.trello.trello.stepdefs;
+
+import com.trello.trello.ui.PageTransporter;
+import com.trello.trello.ui.pages.LoginAtlasianPage;
+import core.selenium.WebDriverHelper;
+import core.selenium.WebDriverManager;
+import com.trello.trello.config.EnvironmentTrello;
+import com.trello.trello.ui.pages.LoginPage;
+import io.cucumber.java.After;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+/**
+ * LoginSteps class.
+ */
+public class LoginSteps {
+
+    private LoginPage login;
+    private LoginAtlasianPage loginAtlasian;
+    /**
+     * Create a new login instance.
+     */
+    @Given("I am on the login page")
+    public void iAmOnTheLoginPage() {
+        PageTransporter.navigateToPage("LOGIN");
+        login = new LoginPage();
+    }
+
+    /**
+     * Login in the two steps.
+     */
+    @When("I set username and password")
+    public void iSetUsernameAndPassword() {
```

Comment:
You should have a parameter in this step or you will have a different login step for each user